cpuminer.go

```diff
@@ -639,29 +616,31 @@ func (m *CPUMiner) GenerateNBlocks(n uint32) ([]*chainhash.Hash, error) {
 		// Create a new block template using the available transactions
 		// in the memory pool as a source of transactions to potentially
 		// include in the block.
-		template, err := m.g.NewBlockTemplate(payToAddr)
+		template, err := m.tg.NewBlockTemplate(payToAddr)
 		m.submitBlockLock.Unlock()
 		if err != nil {
-			errStr := fmt.Sprintf("Failed to create new block "+
-				"template: %v", err)
-			minrLog.Errorf(errStr)
+			minrLog.Errorf("Failed to create new block template: %v", err)
 			continue
 		}
+
 		if template == nil {
-			errStr := fmt.Sprintf("Not enough voters on parent block " +
+			minrLog.Debugf("Not enough voters on parent block " +
 				"and failed to pull parent template")
-			minrLog.Debugf(errStr)
 			continue
 		}
 
+		minrLog.Debugf("new block template from background generator "+
+			"(height: %v, hash: %v)", template.Block.Header.Height,
+			template.Block.BlockHash())
+
 		// Attempt to solve the block.  The function will exit early
 		// with false when conditions that trigger a stale block, so
 		// a new block template can be generated.  When the return is
 		// true a solution was found, so submit the solved block.
-		if m.solveBlock(template.Block, ticker, nil) {
-			block := dcrutil.NewBlock(template.Block)
-			m.submitBlock(block)
-			blockHashes[i] = block.Hash()
+		if m.solveBlock(&template.Block.Header, ticker, nil) {
+			blk := dcrutil.NewBlock(template.Block)
```

Comment:
What's the point of changing these?  It doesn't increase readability nor fix any issue.